.github/workflows/macos-dep-brew.yml

```diff
@@ -0,0 +1,41 @@
+name: MacOS-dep-brew
+
+# https://www.jeffgeerling.com/blog/2020/running-github-actions-workflow-on-schedule-and-other-events
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 2 * * SUN'
+
+jobs:
+  build-macos-dep-brew:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-10.15, macos-11.0]
+
+    steps:
+    # https://github.com/marketplace/actions/cancel-workflow-action
+    - name: Cancel Previous Runs
+      uses: styfle/cancel-workflow-action@0.5.0
+      with:
+        access_token: ${{ github.token }}
+
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Print system information
+      run: sysctl -a | grep machdep.cpu
+
+    - name: Print OS information
+      run: system_profiler SPSoftwareDataType
+
+    - name: Install dependencies
+      run: brew install libpng libjpeg libdc1394 lapack eigen opencv librealsense libxml2 pcl visp
```

Comment:
Before `brew install ...`, I propose to add `brew update; brew upgrade` to test always with the most recent releases